airflow/gcp/hooks/gcs.py

```diff
@@ -273,7 +294,6 @@ def is_updated_after(self, bucket_name, object_name, ts):
     def delete(self, bucket_name, object_name):
         """
         Deletes an object from the bucket.
-
```

Comment:
This change prevents the documentation from being built properly. Can you delete it?